{{cookiecutter.project_name}}/Applications/{{cookiecutter.app_name}}App/Main.cxx

```diff
@@ -22,6 +22,8 @@
 // Slicer includes
 #include "qSlicerApplication.h"
 #include "qSlicerApplicationHelper.h"
+#include "vtkSlicerConfigure.h" // For Slicer_MAIN_PROJECT_APPLICATION_NAME
```

Comment:
Not necessary, as it is included transitively via
 ```
qSlicerApplication.h -> qSlicerCoreApplication.h -> vtkSlicerConfigure.h
```

Reply:
Thanks for the review :pray: 

While it is not strictly required, it is good practice to include headers for classes or macros directly used. It prevents issues if any of the files indirectly included are updated.